ajax/libs/bootstrap-calendar/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "bootstrap-calendar",
+  "version": "0.2.4",
+  "description": "Bootstrap full view calendar",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Serhioromano/bootstrap-calendar.git"
+  },
+  "keywords": [
+    "calendar"
+  ],
+  "author": {
+    "name": "Sergey Romanov"
+  },
+  "license": "BSD",
+  "homepage": "http://bootstrap-calendar.azurewebsites.net/",
```

Comment:
@sufuf3 I can't access this url, can you? We should remove this url if it is broken.